.rubocop.yml

```diff
@@ -1,12 +1,19 @@
+# This configuration only includes the cops that differ from the Rubocop
+# defaults, which can be found here:
+# https://github.com/bbatsov/rubocop/blob/master/config/default.yml
+# https://github.com/bbatsov/rubocop/blob/master/config/enabled.yml
+# https://github.com/bbatsov/rubocop/blob/master/config/disabled.yml
+
 AllCops:
   Include:
     - '**/Gemfile'
     - '**/Rakefile'
   Exclude:
-    - 'bin/**/*'
     - 'db/schema.rb'
-    - 'spec/i18n_spec.rb'
```

Comment:
Or, you can run `rubocop -a spec/i18n_spec.rb` to auto fix it and check it in here if you want